DAO.sol

```diff
@@ -515,60 +506,80 @@ contract DAO is DAOInterface, Token, TokenSale {
             return;
         }
 
-        if (p.amount > actualBalance())
-            throw;
-
         // If the curator removed the recipient from the whitelist, close the proposal
         // in order to free the deposit and allow unblocking of voters
-        if (!allowedRecipients[p.recipient]) {
-            p.open = false;
-            if (!p.creator.send(p.proposalDeposit))
-                throw;
-            sumOfProposalDeposits -= p.proposalDeposit;
+        if (!allowedRecipients[p.recipient] && p.open) {
+            closeProposal(_proposalID);
+            p.creator.send(p.proposalDeposit);
```

Comment:
whether it is send or not, `sumOfProposalDeposits` get reduced in `closeProposal` .